### PR TITLE
whatsapp-for-linux 1.3.1 -> 1.6.1

### PR DESCRIPTION
**Pull request by evil5519**
###### Things done

- Built on platform(s)
  - [X ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/whatsapp-for-linux/default.nix [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 stdenv.mkDerivation rec {
   pname = "whatsapp-for-linux";
-  version = "1.3.1";
+  version = "1.6.1";
 
   src = fetchFromGitHub {
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/whatsapp-for-linux/default.nix:19]: The hash must be updated

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/instant-messengers/whatsapp-for-linux/default.nix (hunk 1 above)
Comment (discite) [reply pkgs/applications/networking/instant-messengers/whatsapp-for-linux/default.nix:19]: Thank you so much, this is my first commit, so let me know if i need to do anything else from my side to see the pull request solved

Comment (Shawn8901) [reply pkgs/applications/networking/instant-messengers/whatsapp-for-linux/default.nix:19]: @discite  you still do not update the hash. Not sure if you have resolved the comment then by accident.
Beside that, there is also a format suggested for the commit messages. That can be read in the contribution guideline (there is a link for that in the PR template).

**Comment by StrictlyMonad** [comment]
Seems reasonable to why #219322 hasn't picked up yet. :joy: 
Want me to close it in favor of your PR?

**Comment by Icy-Thought** [comment]
Previous message was from wrong account, apologize for that.
Want me to close my PR in favour (#219322) of yours?

**Pull request closed without merging**